### PR TITLE
Test seed data #157

### PR DESCRIPTION
**Pull request by maxmcd**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,7 +39,8 @@
   },
   "scripts": {
     "start": "./node_modules/.bin/nodemon server/server.js --ignore node_modules/",
-    "coverage": "istanbul cover _mocha ./test/app/ -- -R spec; open coverage/lcov-report/index.html"
+    "coverage": "istanbul cover _mocha ./test/app/ -- -R spec; open coverage/lcov-report/index.html",
+    "test": "CCENV=testing mocha test/app/"
   },
```
Comment (maxmcd) [inline package.json]: Test now run with `npm test`

File: seeds/seeds.js [hunk 2]
```diff
@@ -1,22 +1,71 @@
...
 exports.seed = function(knex, Promise) {
-  return Promise.join(
-    // Deletes ALL existing entries
-    knex('orgs').del(), 
+  console.log("Running seeds.js".yellow)
+  console.log("Deleting all tables".yellow)
+  return knex.raw(
+    `DROP SCHEMA public CASCADE;
+    CREATE SCHEMA public;`
+  ).then(() => {
```
Comment (maxmcd) [inline seeds/seeds.js]: Just dropping all the tables, a little heavy-handed, want to make sure this isn't ever run on anything important, maybe add a check that we're on a testing database

File: package.json (hunk 1 above)
Comment (kuanb) [inline package.json]: bull

File: seeds/seeds.js (hunk 2 above)
Comment (kuanb) [inline seeds/seeds.js]: I actually think this is not heavy handed in the sense that, each time you run the test, you technically want to build **everything** right? Or is this the wrong way to think about it?

**Comment by kuanb** [comment]
LGTM 

assuming goes here

**Pull request merged**